spec/lib/karafka/connection/topic_consumer_spec.rb

```diff
@@ -43,7 +45,21 @@
 
     it 'expect to use kafka_consumer to get messages and yield' do
       expect(kafka_consumer).to receive(:each_message).and_yield(incoming_message)
-      expect { |block| topic_consumer.fetch_loop(&block) }.to yield_control
+      expect { |block| topic_consumer.fetch_loop(&block) }.to yield_with_args(incoming_message)
+    end
+
+    context 'batch mode' do
```

Comment:
I think we should wrap the test above(46-51 lines) in context as well,  e.g. 'consume each message'.